doc/user/content/sql/alter-cluster.md

```diff
@@ -6,44 +6,159 @@ menu:
     parent: 'commands'
 ---
 
-`ALTER CLUSTER` changes the configuration of a cluster, which **might incur
-downtime**. To rename a cluster, use [`ALTER ... RENAME`](/sql/alter-rename/).
+`ALTER CLUSTER` changes the configuration of a cluster, such as the `SIZE` or
+`REPLICATON FACTOR`.
 
 ## Syntax
 
-#### Set new configuration value
+`ALTER CLUSTER` has the following syntax forms:
 
-{{< diagram "alter-cluster-set.svg" >}}
+{{< tabs >}}
+{{< tab "Set a configuration" >}}
 
-#### Reset configuration to original value
+To set a cluster configuration:
```

Comment:
```suggestion
To change the value of a cluster configuration option:
```

Reply:
The search thing again. Now, having "Change" is also good because people probably would not lookup "Alter" and would instead look up "Change".  But, "set" may be marginally better in that although too generic, people tend to remember some part of the command. 

Once, some of our search results improve, updating the content to "change ..." will definitely be the way to go.